src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -1299,8 +1299,16 @@ namespace ngraph
                 writer << "#pragma omp parallel for\n";
                 writer << "for (size_t i = 0; i < " << out[0].get_size() << "; i++)\n";
                 writer.block_begin();
-                writer << out[0].get_name() << "[i] = (" << result_element_type.c_type_string()
-                       << ")(" << args[0].get_name() << "[i]);\n";
+                writer << out[0].get_name() << "[i] = (";
+                if (result_element_type == element::boolean)
+                {
+                    writer << "bool";
```

Comment:
This would create 2 double quotes. One from line 1302/1311 and one here. is that ok?

Reply:
"" is used to pass string to writer. "" in line 1302/1311 is not related to "" here.

Reply:
Thanks for the clarifications.